src/modules/AllWidgets.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+
+namespace WPML\PB\Elementor\Modules;
```

Comment:
@GeorgeBotsev, we have a dedicated folder for media stuff: `src/media/modules` and so the namespace should be `WPML\PB\Elementor\Media\Modules`.

Reply:
The class is now moved there as requested.